da/celestia/celestia.go

```diff
@@ -629,3 +604,46 @@ func (c *DataAvailabilityLayerClient) getDataAvailabilityHeaders(height uint64)
 
 	return headers.DAH, nil
 }
+
+// Celestia syncing in background
+func (c *DataAvailabilityLayerClient) syncCelestia(rpc *openrpc.Client) {
```

Comment:
we're already in celestia so maybe just name the func `sync`?

Reply:
changed